lib/migrate.js

```diff
@@ -741,7 +741,8 @@ module.exports = {
                 if (index < migrationCommands.length)
                 {
                     let command = migrationCommands[index];
-                    console.log("[#"+index+"] execute: " + command.fn);
+                    const tableName = (command.params && command.params[0]) || '<no table name>'
```

Comment:
This project already depends on `lodash`. You could consider doing this instead:
```js
const tableName = _.get(command, 'params[0]', '<no table name>');
```

Reply:
@PaulMest thanks, I considered and in this specific case I feel like it comes down to personal preference, I don't see strong advantage of one or the other approach. For fields that are 3+ levels deep I think _.get has significant advantages.